### PR TITLE
Fix do not disturb switch

### PR DESCRIPTION
**Pull request by DurgNomis-drol**
Small PR to fix switch not working.

I also moved JSON attribute to a CONST.

@leikoilja Any thing else from the other PR review? 🤔 

@KapJI About it blinking I think i know how to fix that. but will make a separate PR for that. 😄

### CODE REVIEW

**Comment by KapJI** [comment]
There was also a README change 🙂

File: custom_components/google_home/const.py [hunk 1]
```diff
@@ -84,6 +84,7 @@
 # JSON parameter values when retrieving information from devices
 JSON_ALARM = "alarm"
 JSON_TIMER = "timer"
+JSON_NOTIFICATION_ENABLED = "notifications_enabled"
```
Comment (KapJI) [inline custom_components/google_home/const.py:87]:
```suggestion
JSON_NOTIFICATIONS_ENABLED = "notifications_enabled"
```

**Review by KapJI — APPROVED** [review]

**Comment by DurgNomis-drol** [comment]
Thanks @KapJI for approving it 😃

File: custom_components/google_home/api.py [hunk 2]
```diff
@@ -332,33 +333,34 @@ async def update_do_not_disturb(
 
         data = None
 
-        if enable is None:
+        if enable is not None:
```
Comment (leikoilja) [inline custom_components/google_home/api.py:336]: This could just be `if enable`, right? :)

Comment (DurgNomis-drol) [reply custom_components/google_home/api.py:336]: I tried that, but if the value is false, python just moves to else instead. 😊

Comment (leikoilja) [reply custom_components/google_home/api.py:336]: hm... but do we expect `True` here? Maybe then `if enable is True`? 😅

Comment (DurgNomis-drol) [reply custom_components/google_home/api.py:336]: We expected True or False if we want to set the status on the device or None if we just want get status from device 😊

Comment (leikoilja) [reply custom_components/google_home/api.py:336]: ouuu gotcha now. Thanks for explaining 👍

Comment (DurgNomis-drol) [reply custom_components/google_home/api.py:336]: No problem 😊

**Pull request merged**